framework/src/application/node/rebuilder.js

```diff
@@ -23,36 +23,36 @@ class Rebuilder {
 		// components
 		channel,
 		logger,
-		storage,
 		// Unique requirements
 		genesisBlock,
 		// Modules
 		processorModule,
 		blocksModule,
 		// Constants
 		activeDelegates,
+		storage,
 	}) {
 		this.isActive = false;
 		this.isCleaning = false;
 
 		this.channel = channel;
 		this.logger = logger;
-		this.storage = storage;
 		this.genesisBlock = genesisBlock;
 
 		this.processorModule = processorModule;
 		this.blocksModule = blocksModule;
 		this.constants = {
 			activeDelegates,
 		};
+		this.storage = storage;
```

Comment:
we can remove storage?

Reply:
We can't remove it yet because there is a call to `Block.delete`

Reply:
ah maybe we can add the function in the `lisk-blocks` too

Reply:
Will we do writes in separate PR?

Reply:
I think this is the only one that is missing. all other things are already solved